projects.rst

```diff
@@ -1,6 +1,6 @@
-**************
-Nengo projects
-**************
+***************
+Nengo ecosystem
```

Comment:
I have to admit I'm not super sold on using the word "ecosystem" in so many places, but I agree that "projects" is too general a word for what this page shows...

The only real alternative I can think of is "Nengo packages," which is somewhat Python specific terminology, but is more specific than projects. If people don't like that, then I'm fine with "ecosystem".